src/modules/ekf2/EKF/common.h

```diff
@@ -345,7 +353,7 @@ struct parameters {
 	float mag_heading_noise{3.0e-1f};       ///< measurement noise used for simple heading fusion (rad)
 
 #if defined(CONFIG_EKF2_MAGNETOMETER)
-	float mag_delay_ms{0.0f};               ///< magnetometer measurement delay relative to the IMU (mSec)
+	float mag_delay_ms {0.0f};              ///< magnetometer measurement delay relative to the IMU (mSec)
```

Comment:
is it always the first one after a `#if defined` that needs an extra space?

Reply:
Newer astyle may have fixed it, I'd like to get the container updated soon and shift the default CI environment to Ubuntu 24.04 (latest LTS).